data/sql_incremental_aggregates/prepare_schedule_a_aggregate_contributor.sql

```diff
@@ -17,12 +17,14 @@ and (memo_cd != 'X' or memo_cd is null)
 group by cmte_id, cycle, clean_repeated(contbr_id, cmte_id)
 ;
 
+alter table ofec_sched_a_aggregate_contributor add column idx serial primary key;
+
 -- Create indices on aggregate
-create index on ofec_sched_a_aggregate_contributor (cmte_id);
-create index on ofec_sched_a_aggregate_contributor (cycle);
-create index on ofec_sched_a_aggregate_contributor (contbr_id);
-create index on ofec_sched_a_aggregate_contributor (total);
-create index on ofec_sched_a_aggregate_contributor (count);
+create index on ofec_sched_a_aggregate_contributor (cmte_id, idx);
```

Comment:
Clearly, creating these multi-column indexes seems central to the strategy here. I am curious, though, whether you also tried letting Postgres combine single-column indexes for each key + `idx`. My only concern is that these indexes could be getting quite large... have you looked at how much memory each of these take (vs. working memory on the RDS instance)? If the indexes get large enough, it could limit Postgres' ability to keep frequently-used indexes in the cache. But if Postgres can effectively use single-column indexes in combination, it might be slightly slower per-query, but your average time could be faster, since you're less likely to get hit with the swap time necessary for Postgres to load up the index(es) it needs.